root/programs/C#/Frameworks/Tools/DPQuery_Tool/DPQuery_Tool.csproj

```diff
@@ -83,6 +83,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\Infrastructure\Build\Oracle.DataAccess.dll</HintPath>
```

Comment:
@daisukenishino 

I think this `Oracle.DataAccess.dll` library is unnecessary.